### PR TITLE
Add mg to contribs

### PR DESCRIPTION
**Pull request by segfault**
adding myself to the contributor list

### CODE REVIEW

**Comment by jasnell** [comment]
@segfault ... Really just a formality, but before adding to contributors, you'll need a formal nomination to be added as a @covidgreen/members from another existing member, posted as an issue with no objections from other members. See https://github.com/covidgreen/covid-green-backend-api/issues/7 as an example. This PR can't land until that is done. Once you've gone through the nomination, I can add you to the members team. Will mark this PR "changes required" until that goes through.

**Review by jasnell — CHANGES_REQUESTED** [review]
Pending @covidgreen/members nomination

**Comment by jasnell** [comment]
Nomination issue: https://github.com/covidgreen/covid-green-infra/pull/2

**Review by dgonzalez — APPROVED** [review]

**Comment by dgonzalez** [comment]
This is +1 from me as well

File: monitoring.tf [hunk 1]
```diff
@@ -1,197 +0,0 @@
-# #########################################
```
Comment (jasnell) [inline monitoring.tf:1]: Unrelated change?

Comment (dgonzalez) [reply monitoring.tf:1]: no, it is good.

**Comment by dgonzalez** [comment]
Welcome!

File: monitoring.tf (hunk 1 above)
Comment (dgonzalez) [reply monitoring.tf:1]: no, it is good.

**Pull request merged**